jenkins/bootstrap.py

```diff
@@ -343,6 +343,7 @@ def finish(gsutil, paths, success, artifacts, build, version, repo):
 
     if os.path.isdir(artifacts) and any(f for _, _, f in os.walk(artifacts)):
         try:
+            call(['sudo', 'chmod', 'a+rX', '-R', artifacts])
```

Comment:
We should not run sudo commands in bootstrap. This also seems specific to kubernetes. Will you move this to job.sh or dockerized-e2e-runner.sh?